snippets/remote_execution_ssh_keys.erb

```diff
@@ -27,6 +32,12 @@
 user_exists=false
 getent passwd <%= ssh_user %> >/dev/null 2>&1 && user_exists=true
 
+<% if ssh_user != 'root' && @host.params['remote_execution_create_user'] -%>
```

Comment:
I used a global parameter set to true, so it is not actually a boolean but I used it as one. 
But I am open for other suggestions, if you prefer some other handling.